ISL.Reidentification.Core.Tests.Unit/Services/Orchestrations/Identifications/IdentificationOrchestrationTests.Logic.cs

```diff
@@ -0,0 +1,81 @@
+﻿// ---------------------------------------------------------
+// Copyright (c) North East London ICB. All rights reserved.
+// ---------------------------------------------------------
+
+using System;
+using System.Threading.Tasks;
+using FluentAssertions;
+using ISL.ReIdentification.Core.Models.Foundations.AccessAudits;
+using ISL.ReIdentification.Core.Models.Foundations.ReIdentifications;
+using Moq;
+
+namespace ISL.ReIdentification.Core.Tests.Unit.Services.Orchestrations.Identifications
+{
+    public partial class IdentificationOrchestrationTests
+    {
+        [Fact]
+        public async Task ShouldCreateAuditAccessRecordIfRequestDoesNotHaveAccessAsync()
+        {
+            // given
+            DateTimeOffset someDateTimeOffset = GetRandomDateTimeOffset();
+            Guid someGuid = Guid.NewGuid();
+            IdentificationRequest someIdentificationRequest = CreateRandomIdentificationRequest(false);
+            IdentificationRequest inputIdentificationRequest = someIdentificationRequest;
+            IdentificationRequest expectedIdentificationRequest = inputIdentificationRequest;
```

Comment:
Dereference to that you can ensure nothing has changed the state.  The test should fail when you do this
```suggestion
            IdentificationRequest expectedIdentificationRequest = inputIdentificationRequest.DeepClone();
```